.github/workflows/quality.yml

```diff
@@ -111,3 +111,15 @@ jobs:
         with:
           name: report
           path: apps/web/playwright-report/
+
+      - name: Save PR number
+        env:
+          PR_NUMBER: ${{ github.event.number }}
+        run: |
+          echo $PR_NUMBER > prNumber
```

Comment:
is this the only way to know the prnumber

im an actions noob but seems weird

Reply:
apparently so, I logged all the context values in this run and the pr number is no where to be seen
https://github.com/billc-dev/typehero/actions/runs/6263905130/job/17009349805

https://github.com/orgs/community/discussions/25220#discussioncomment-3246922